rust/ethers-prometheus/src/lib.rs

```diff
@@ -478,17 +494,12 @@ impl<M: Middleware> Debug for PrometheusMiddleware<M> {
     }
 }
 
-/// Get the metrics appropriate chain name from the chain ID.
-pub fn metrics_chain_name(chain_id: Option<u64>) -> String {
-    if let Some(chain_id) = chain_id {
-        if let Ok(chain) = Chain::try_from(chain_id) {
-            format!("{chain}")
-        } else {
-            format!("{chain_id}")
-        }
-    } else {
-        "unknown".into()
-    }
+/// Uniform way to name the chain.
+fn chain_name(chain: &Option<ChainInfo>) -> &str {
```

Comment:
Why can't this be defined on `ChainInfo`? This is far more verbose than I would expect, but maybe that's just the way rust is..

Reply:
Because I was treating ChainInfo like a C struct in my brain and this more closely mirrored the old code. Yes, it should be a method.

Reply:
Oh, actually, because ChainInfo being None is a good case to handle as well, so it will be a static member which at that point it's basically the same as what I did here.

Reply:
What about a method of `PrometheusMiddlewareConf`?